### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by troch**
Update examples for v3.

### CODE REVIEW

File: examples/_template/rxjs/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "react": "16.4.1",
         "react-dom": "16.4.1",
         "react-scripts": "1.0.10",
-        "refract-rxjs": "~1.0.0-4",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/_template/rxjs/package.json]: 🤔

File: examples/counter/rxjs/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
         "react": "~16.5.2",
         "react-dom": "~16.5.2",
         "react-scripts": "~1.1.5",
-        "refract-rxjs": "~2.2.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/counter/rxjs/package.json]: 🤔

File: examples/debounced-fetch/rxjs/package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
         "react": "16.1.1",
         "react-dom": "16.1.1",
         "react-scripts": "1.0.10",
-        "refract-rxjs": "~1.0.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/debounced-fetch/rxjs/package.json]: 🤔 Did you test these? 😁

File: examples/field-validation/rxjs/package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
         "react": "16.1.1",
         "react-dom": "16.1.1",
         "react-scripts": "1.0.10",
-        "refract-rxjs": "~1.0.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/field-validation/rxjs/package.json]: 🤔

File: examples/redux-fetch/rxjs/package.json [hunk 5]
```diff
@@ -8,7 +8,7 @@
         "redux": "~4.0.0",
         "redux-devtools-extension": "~2.13.5",
         "refract-redux-rxjs": "~1.0.0",
-        "refract-rxjs": "~1.0.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/redux-fetch/rxjs/package.json]: 🤔

File: examples/routing/rxjs/package.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
         "react": "16.1.1",
         "react-dom": "16.1.1",
         "react-scripts": "1.0.10",
-        "refract-rxjs": "~1.0.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/routing/rxjs/package.json]: 🤔

File: examples/typeahead/rxjs/package.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
         "react": "16.4.1",
         "react-dom": "16.4.1",
         "react-scripts": "1.0.10",
-        "refract-rxjs": "~1.0.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/typeahead/rxjs/package.json]: 🤔

File: examples/visit-time/rxjs/package.json [hunk 8]
```diff
@@ -5,7 +5,7 @@
         "react-dom": "16.4.1",
         "react-scripts": "1.0.10",
         "react-state-hoc": "~3.0.0",
-        "refract-rxjs": "~1.0.0",
+        "refract-xjs": "3.0.0",
```
Comment (jtmcgrath) [inline examples/visit-time/rxjs/package.json]: 🤔

File: examples/debounced-fetch/rxjs/package.json (hunk 3 above)
Comment (troch) [reply examples/debounced-fetch/rxjs/package.json]: Nope, was planning to test and merge once 3.0.0 is out

**Comment by troch** [comment]
Will merge once released

**Pull request merged**